MAUI/DataGrid/conditional-styling.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: Conditional Styling in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about Conditional Styling support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Conditional Styling in .NET MAUI DataGrid (SfDataGrid)
+The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows to customize the style of the individual cells and rows based on the requirements. 
+
+## Conditional row style
+The data rows can be customized conditionally by writing the style with converter for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its BindingContext is underlying datasource object.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <local:ColorConverter x:Key="converter"/>
+        <Style TargetType="syncfusion:DataGridRow">
+            <Setter Property="Background" Value="{Binding Converter={StaticResource converter}}" />
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
+{% endhighlight %}
+{% highlight xaml tabtitle="ColorConverter.cs"%}
+public class ColorConverter : IValueConverter
+{
+    object IValueConverter.Convert(object value, Type targetType, object parameter, CultureInfo info)
+    {
+        var input = (value as OrderInfo).OrderID;
+        if (input < 1003)
+            return Colors.Bisque;
+        else if (input < 1007)
+            return Colors.LightBlue;
+        else
+            return Colors.White;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle.png)
+
+## How to style a particular row based on row index
+Styling can be applied to a particular row based on RowIndex property by writing the style for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.
```

Comment:
Styling can be applied to a particular row based on the RowIndex property by writing the style for the [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.